{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -24,6 +23,7 @@ gunicorn = "^19.9"
 python-decouple = "^3.1"
 dump-env = "^0.2"
 bcrypt = "^3.1"
+django = "^2.2"
```

Comment:
Should be on top

Reply:
> Should be on top

fix
But I think order will not be important.